charts/astronomer/templates/houston/houston-configmap.yaml

```diff
@@ -197,96 +216,80 @@ data:
               except Exception as e:
                 pod_mutation_hook_deprecated(pod)
 
-          resources:
-            limits:
-              ephemeral-storage: "2Gi"
-            requests:
-              ephemeral-storage: "1Gi"
-
-        # Redis settings
-        redis:
-          # This is here for upgrading to 0.11.2+ of Airflow.
-          safeToEvict: true
-
-        # Pgbouncer settings
-        pgbouncer:
-          # Pgbouner pod disruption budget
-          podDisruptionBudget:
-            enabled: false
-
-            # PDB configuration
-            config:
-              maxUnavailable: 1
-
-        # Default quotas for airflow deployments.
-        quotas:
-          requests.ephemeral-storage: "50Gi"
-          limits.ephemeral-storage: "50Gi"
-
-        # This is here for backward compatibility.
-        # 0.11+ versions of Airflow have this defined
-        # so this needs to be passed through to all upgrades.
-        podMutation:
-          tolerations: []
-
-        # This is here for upgrading to 0.11.0+ of Airflow.
-        defaultAirflowRepository: quay.io/astronomer/ap-airflow
-
-        # This is here for upgrading to 0.11.1+ of Airflow.
-        data:
-          metadataConnection:
-            sslmode: disable
-          resultBackendConnection:
-            sslmode: disable
-
-    {{- if not .Values.global.singleNamespace }}
-        # Enable cleanup CronJob in Airflow deployents
-        # to cleanup evicted/failed/succeeded pods.
-        cleanup:
-          enabled: true
-          schedule: "*/15 * * * *"
-    {{- end }}
-
-        # Disable the postgres subchart. The platform pass credentials down
-        # from the platform configuration.
-        postgresql:
-          enabled: false
 
-    {{- if .Values.global.sccEnabled }}
-        # If security context constraints are enabled, enable SCCs for airflow-chart
-        sccEnabled: true
-    {{- end }}
+          # Redis settings
+          redis:
+            # This is here for upgrading to 0.11.2+ of Airflow.
```

Comment:
Is this still needed?

Reply:
same as https://github.com/astronomer/astronomer/pull/1135/files#r712392264

Reply:
```suggestion
            # This is here for upgrading to 0.11.2+ of airflow-chart
```